apps/angular/injection-token/src/app/timer.component.ts

```diff
@@ -1,13 +1,15 @@
-import { Component } from '@angular/core';
+import { Component, Inject } from '@angular/core';
 import { toSignal } from '@angular/core/rxjs-interop';
 import { interval } from 'rxjs';
-import { DEFAULT_TIMER } from './data';
+import { TIMER_TOKEN } from './data';
+//import { DEFAULT_TIMER } from './data';
 
 @Component({
   selector: 'timer',
   standalone: true,
   template: ` Timer running {{ timer() }} `,
 })
 export class TimerComponent {
-  timer = toSignal(interval(DEFAULT_TIMER));
+  constructor(@Inject(TIMER_TOKEN) public time: number) {}
```

Comment:
perfer the `inject` function than the decorator. This way the property is properly typed